### PR TITLE
adding mkl-service recipe

### PR DESCRIPTION
**Pull request by oleksandr-pavlyk**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages
- [X] Build number is 0
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

@dmitrii-zagornyi please confirm that you are OK to be a maintainer of this package.

### CODE REVIEW

File: recipes/mkl-service/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+requirements:
+    build:
+      - {{ compiler('c') }}
+    host:
+      - python
+      - setuptools
+      - mkl-devel
+      - cython
+      - numpy 1.11.*
+      - six
```
Comment (chrisburr) [inline recipes/mkl-service/meta.yaml]: I don't think `six` is needed in the `host` requirements?
```suggestion
```
(`host` should only contain dependencies of the `setup.py` script.)

File: recipes/mkl-service/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{% set version = "2.0.1" %}
+{% set buildnumber = 0 %}
```
Comment (chrisburr) [inline recipes/mkl-service/meta.yaml]: If a variable is only used once its better not to have it to give the update bot a better chance at parsing it correctly.
```suggestion
```

File: recipes/mkl-service/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+package:
+    name: mkl-service
+    version: {{ version }}
```
Comment (chrisburr) [inline recipes/mkl-service/meta.yaml]:
```suggestion
    version: 0
```

File: recipes/mkl-service/bld.bat [hunk 4]
```diff
@@ -0,0 +1,4 @@
+@rem Remember to activate Intel Compiler, or remoe these two lines to ise Microsoft Visual Studio compiler
```
Comment (chrisburr) [inline recipes/mkl-service/bld.bat]:
```suggestion
@rem Remember to activate Intel Compiler, or remove these two lines to use Microsoft Visual Studio compiler
```

File: recipes/mkl-service/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    commands:
+      - nosetests -v mkl
+    requires:
+      - nose
+    imports:
+      - mkl
+
+about:
+    home: http://github.com/IntelPython/mkl-service
+    license: BSD 3-Clause
```
Comment (chrisburr) [inline recipes/mkl-service/meta.yaml]: From https://opensource.org/licenses/alphabetical:
```suggestion
    license: BSD-3-Clause
    license_family: BSD
```

File: recipes/mkl-service/meta.yaml (hunk 1 above)
Comment (oleksandr-pavlyk) [reply recipes/mkl-service/meta.yaml]: Thanks @chrisburr , I removed the line, as well as corrected `license` and added `license-family`. 

I would like to keep using jinja variables. I like the fact that all I need to change when updating the recipe is atop of the `meta.yaml`.

**Comment by oleksandr-pavlyk** [comment]
Dmitrii has agreed to be a maintainer, and posted a comment to that effect:

image

It does not show on the feed at the moment due to github having erroneously flagged his account.

**Comment by chrisburr** [comment]
I'm not sure how conda-forge's scripts will handle an invalid user ID so I'll remove them for now and they can be readded in the feedstock at any time.

**Pull request merged**